### PR TITLE
Release notes for 0.5.1

### PR DESCRIPTION
**Pull request by cPu1**

### CODE REVIEW

File: docs/release_notes/0.5.1.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Release 0.5.1
+
+## Features
+
+- Add support for Kubernetes 1.14 (#1281)
+
+## Improvements
```
Comment (errordeveloper) [inline docs/release_notes/0.5.1.md]: Could you please add:
`- use AWS SDK v1.23.15 (#1278)`

File: docs/release_notes/0.5.1.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Release 0.5.1
+
+## Features
+
+- Add support for Kubernetes 1.14 (#1281)
```
Comment (errordeveloper) [inline docs/release_notes/0.5.1.md]: I thought we said that items begin with lower-case, which is what I used in 0.5.0 notes...

**Comment by errordeveloper** [comment]
CI seems to fail (#1287), I'll re-run it.

**Comment by errordeveloper** [comment]
By the way, I think this PR should had been against master and then cherry-picked to `release-0.5.1`. No need to redo it, but do make sure to cherry-pick onto master once merged.

File: docs/release_notes/0.5.1.md (hunk 2 above)
Comment (cPu1) [reply docs/release_notes/0.5.1.md]: The 0.5.0 notes are not in master. I looked at the 0.5.0-rc.0.md and thought we've started using capitalised words to start items.

**Comment by cPu1** [comment]
>By the way, I think this PR should had been against master and then cherry-picked to release-0.5.1. 

It _is_ against master. I'll cherry-pick it to release-0.5.1.

File: docs/release_notes/0.5.1.md (hunk 2 above)
Comment (cPu1) [reply docs/release_notes/0.5.1.md]: I updated the notes to use lowercase letters.

**Comment by errordeveloper** [comment]
ok, I see, branch name is "release-0.5.1", that's what got me confused :)

**Pull request merged**